integrations/dialpad/types.ts

```diff
@@ -0,0 +1,33 @@
+export interface EmailEntity {
+    email: string;
+}
+
+export interface SuccessResponse {
+    success: boolean;
+}
+
+export interface User {
+    id: string;
+    email: string;
+    firstName: string;
+    lastName: string;
+}
+
+export interface DialpadUser {
```

Comment:
The third party api has a richer response than this. 

```
{
      "admin_office_ids": [
        "124"
      ],
      "company_id": "123",
      "country": "us",
      "date_active": "2021-06-20T19:18:00",
      "date_added": "2021-06-20T19:18:00",
      "date_first_login": "2021-06-20T19:18:00",
      "do_not_disturb": false,
      "emails": [
        "bot@fuzz-ball.com"
      ],
      "extension": "20000",
      "first_name": "テÉst",
      "forwarding_numbers": [
        "+14152301358"
      ],
      "group_details": [
        {
          "do_not_disturb": false,
          "group_id": "124",
          "group_type": "office",
          "role": "admin"
        }
      ],
      "id": "2",
      "image_url": "https://dialpad.example.com/avatar/user/ag5kZXZ-dWJlci12b2ljZXIRCxILVXNlclByb2ZpbGUYAgw.png?version=36c0561fb8e0f5765ccfb3a5316f6d5d",
      "is_admin": true,
      "is_available": true,
      "is_on_duty": false,
      "is_online": false,
      "is_super_admin": true,
      "job_title": "Mock Job Title",
      "language": "en",
      "last_name": "Bot",
      "license": "talk",
      "location": "Mock Location",
      "muted": false,
      "office_id": "124",
      "phone_numbers": [
        "+14155551001"
      ],
      "state": "active",
      "status_message": "Mock Status",
      "timezone": "US/Pacific"
    }
```

The type here should reflect the documented response from dialpad